irohad/ametsuchi/impl/postgres_query_executor.cpp

```diff
@@ -929,5 +972,21 @@ namespace iroha {
           std::move(response_txs), query_hash_);
     }
 
+    template <typename ReturnValueType>
+    bool PostgresQueryExecutorVisitor::existsInDb(
+        const std::string &table_name,
+        const std::string &key_name,
+        const std::string &value_name,
+        const std::string &value) const {
+      auto cmd = (boost::format(R"(SELECT %s
```

Comment:
Please use data binding with `:value_name`, e.g. `SELECT :value_name` and `soci::use(value_name)` instead of raw format string.

Reply:
As it seems, is is at least not possible to bind the name of the table to be selected from, and using both binding and boost::variant looks like a bad idea, so I decided to pick the boost